scripts/socketplane.sh

```diff
@@ -316,6 +320,12 @@ start_socketplane() {
 
     mkdir -p /var/run/socketplane
     echo $cid > /var/run/socketplane/cid
+
+    if [ "$ps" = "yes" ]; then
+	pscid=$(docker run -d --name powerstrip -v /var/run/docker.sock:/var/run/docker.sock \
+	    -v /etc/socketplane/adapters.yml:/etc/powerstrip/adapters.yml --net=host  \
```

Comment:
This seems specific to the vagrant setup. Shouldnt this be applicable to non-vagrant setup as well ?